src/cli.rs

```diff
@@ -20,18 +20,18 @@ pub struct Rover {
 pub enum Command {
     ///  ⚙️  Manage configuration
     Config(command::Config),
-    ///  🧱 Fetch a schema
+    ///  🧱  Work with a monolitic graph
```

Comment:
```suggestion
    ///  🧱  Work with a monolithic graph
```

is this the term? or should we say "non-federated" (could be confusing if you don't know what federation is).. just spitballing here

Reply:
Yeah, that makes sense!